bolt-modules/prompt/spec/functions/prompt_spec.rb

```diff
@@ -28,6 +28,20 @@
     expect(result.unwrap).to eq(response)
   end
 
+  it 'returns a default value if no input is provided' do
```

Comment:
A test to verify wrapping + not printing if `:sensitive` is provided would also be good, since that'd be a pretty bad bug.